int32 robotID
int32 severity
string text
int32 userID	# -1 --> everyone should speak, 0 --> no one should speak
string speakMsg	# text to be spoken
